app/controllers/jobs_controller.rb

```diff
@@ -1,7 +1,7 @@
-require 'autoConfig'
-
 class JobsController < ApplicationController
 
+  autolabRequire File.join(Rails.root, 'config/autogradeConfig.rb')
```

Comment:
This seems to miss the point of using join. I'd use 'Rails.root.join("config", "autogradeConfig.rb")'